plugins/frontend/backstage-plugin-github-pull-requests/src/components/Home/RequestedReviewsCard/Content.tsx

```diff
@@ -26,10 +26,15 @@ import {
 } from '../../useGithubLoggedIn';
 import Alert from '@material-ui/lab/Alert';
 
-const RequestedReviewsContet = () => {
-  const { loading, error, value } = useGithubSearchPullRequest(
-    `is:open is:pr review-requested:@me archived:false`,
-  );
+type RequestedReviewsCardProps = {
+  query: string;
```

Comment:
Ah, I guess this is meant to be like so?
```suggestion
  query?: string;
```